surveys/promise-rejections/survey.md

```diff
@@ -0,0 +1,80 @@
+# Node.js Promise reject use case survey
+
+## Are you currently using Promises, async/await, a mix, or neither?
+
+Assume using as creating your own `new Promise`, `Promise.resolve`, and `async function`; or using libraries that produce Promises (that you use with `.then`/`.catch`, or with `.await`)
+
+(Multiple choice)
+
+  - [ ] Promises
+  - [ ] async/await
+  - [ ] Not writing Promise-based code or using Promise-based libraries
+
+## How are you handling rejections today?
+
+When consuming Promises, async functions or thenables, which of the options below do you use to handle rejections?
+
+(Multiple choice)
+
+  - [ ] `.catch()`
+  - [ ] `try / catch` wrapping an `await` operation
+  - [ ] Ignore the rejection
+  - [ ] Not writing Promise-based code or using Promise-based libraries
+
+## Do you use `process.on('unhandledRejection')` as a global handler for unhandled rejections?
+ 
+  - [ ] Yes
+  - [ ] No
+
+## Do you use the `--unhandled-rejections` command-line flag as a global handler for unhandled rejections?
+
+  - [ ] Yes
+  - [ ] No
+
+## Are you using Promises in customer facing, production applications?
+
+  - [ ] Yes
+  - [ ] No
+  - [ ] I don't know
+
+## Which of the use cases described below do you spend most of your time on?
+
+  - [ ] Third-party libraries
+  - [ ] Command-line tools
+  - [ ] Web servers
+  - [ ] Computation-heavy processing
+  - [ ] Desktop applications
+  - [ ] Other (please elaborate)
+
+## When a rejected promise doesn't have a catch handler, what is the behavior of your application today? 
+
+  - [ ] Node.js logs a warning alongside a deprecation notice, execution continues
+  - [ ] Logs a warning, continue running, no deprecation warning
+  - [ ] Logs a warning, continue running, no deprecation warning, exits with an error code when program finishes
+  - [ ] Exit as soon as possible
+  - [ ] Other (please elaborate)
+
+## What should be the default Node.js behavior for unhandled rejections?
```

Comment:
Like above, I don't think people understand what unhandled rejections are very well.

I would prefer "when you throw an error from an async function and don't `.catch` it" or something similar that describes that the issue being discussed is errors thrown and not just `Promise.reject` calls. A lot of people don't know that both of these create rejections.